### PR TITLE
PYTHON-4680 Make llama-index use local atlas

### PR DESCRIPTION
**Pull request by Jibola**

### CODE REVIEW

File: llama-index-python-kvstore/run.sh
Comment (caseyclements) [inline llama-index-python-kvstore/run.sh:21]: This instead of closing Shruti's branch and opening up another PR? Or maybe commenting on her PR?

**Review by caseyclements — CHANGES_REQUESTED** [review]

File: llama-index-python-kvstore/run.sh
Comment (Jibola) [reply llama-index-python-kvstore/run.sh:21]: I'll add the JIRA ticket, but think this is a good temporary stop-gap since waiting on Shruti's PR for approval is an external dependency.

File: llama-index-python-kvstore/run.sh
Comment (Jibola) [inline llama-index-python-kvstore/run.sh:20]:
```suggestion
# PYTHON-4522: Will fix requirement in llama-index repo
```

**Review by caseyclements — APPROVED** [review]
LGTM

**Pull request merged**
